rewatch/src/build/packages.rs

```diff
@@ -582,6 +579,13 @@ fn extend_with_children(
     build
 }
 
+#[derive(Clone, Copy)]
+pub enum DevDeps {
+    Build,
+    DontBuild,
+    Clean,
```

Comment:
I don't understand this variant. We want to know if we need to build dev deps or not. Cleaning is a separate task, not a property of we need to build dev deps or not. Whether to clean can be derived from if we want to build or not (if we build we also need to clean). Or am I misunderstanding this?